doc/releases/changelog-dev.md

```diff
@@ -167,7 +167,7 @@ array([False, False])
 * `transmon_drive` is updated in accordance with [1904.06560](https://arxiv.org/abs/1904.06560). In particular, the functional form has been changed from $\Omega(t)(\cos(\omega_d t + \phi) X - \sin(\omega_d t + \phi) Y)$ to $\Omega(t) \sin(\omega_d t + \phi) Y$.
   [(#4418)](https://github.com/PennyLaneAI/pennylane/pull/4418/)
 
-* `draw_mpl` accepts `style='pennylane'` to draw PennyLane-style circuit diagrams, and `style.use` in `matplotlib.pyplot` accepts `pennylane` to create PennyLane-style plots. [(#3950)](https://github.com/PennyLaneAI/pennylane/pull/3950)
+* `draw_mpl` accepts `style='pennylane'` to draw PennyLane-style circuit diagrams, and `style.use` in `matplotlib.pyplot` accepts `pennylane.drawer.plot` to create PennyLane-style plots. [(#3950)](https://github.com/PennyLaneAI/pennylane/pull/3950)
```

Comment:
ahh, that is a much longer thing to type only to satisfy my file organization... I might regret my decision now that I'm seeing it written out. maybe don't move it back just yet, but I'll let others weigh in on what they would prefer.

Reply:
oh I see your other comment, it should have been `pennylane.plot` before so it just adds one more thing. either way, one more is sometimes one too many so I'm still unsure

Reply:
You're totally right, I made a mistake that first time around. Yep, this is between `pennylane.plot` and `pennylane.drawer.plot`, depending on if it's nested one folder deeper or not. No way to avoid the 'plot' part (name of the style sheet) and no way to avoid the 'pennylane' part (need to refer to the package).

Reply:
@ikurecic up to you but you could also add the image here.

Reply:
I think it's fine without, but I'm adding a note about the font right here.